flora.cabal

```diff
@@ -195,7 +195,7 @@ library flora-test-fixtures
     Flora.PackageFixtures
     Flora.UserFixtures
 
-test-suite flora-test
+executable flora-test
```

Comment:
hmmm, I don't know what to think of this :-P

Reply:
I told you it'd make your eyes bleed :grin:

Reply:
Now it gets copied over to the bin folder by the generic builder, so we can run it in test.nix, furthermore the suite get's disabled for the generic builder as well (which is handy because now we don't need to figure out how to deal with the generic builders configuration).